app/src/main/res/layout/fragment_home.xml

```diff
@@ -40,12 +40,21 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
-            <Button
+
+        <TextView
+            android:id="@+id/top5transTextView"
+            android:layout_width="358dp"
+            android:layout_height="371dp"
+            android:layout_marginTop="10dp"
+            android:layout_marginLeft="10dp"
+            android:text="TextView" />
```

Comment:
Get rid of it because this is for testing purposes. We don't need this for our app